synopsys-task/src/synopsys-task/download-tool.ts

```diff
@@ -67,7 +69,16 @@ export async function downloadTool(
         tl.debug(
           `Failed to download "${fileName}" from "${url}". Code(${response.message.statusCode}) Message(${response.message.statusMessage})`
         );
-        reject(new Error(response.message.statusCode?.toString()));
+        reject(
```

Comment:
May be we can check with or inform Tester and Raj regarding the changed message of error. If we are logging it anywhere.
Same we can check for other places as well, if we are logging anywhere

Reply:
@jahid1209 please address this, since these are the changes for return_status.

Reply:
@kishorikumar We will inform the tester about that.